config/crd/bases/anywhere.eks.amazonaws.com_cloudstackmachineconfigs.yaml

```diff
@@ -57,12 +57,10 @@ spec:
                   be specified as a UUID or name
                 properties:
                   id:
-                    description: Id of a resource in the CloudStack environment. Mutually
-                      exclusive with Name
+                    description: Id of a resource in the CloudStack environment.
```

Comment:
Are these 2 properties no longer mutually exclusive?

Reply:
they never were mutually exclusive. After we changed the field from being ambiguous, they could both be set, and Id would simply take priority. The documentation was outdated

Reply:
How do we communicate the priority of the fields to consumers?